bin/wnews.dart

```diff
@@ -41,13 +68,18 @@ void main(List<String> arguments) async {
     exit(0);
   }
 
+  final now = DateTime.now();
+  final today = DateFormat('yyyy/MM/dd').format(now);
+
   final client = HttpClient();
-  final request = await client
-      .getUrl(Uri.parse('https://en.m.wikipedia.org/wiki/Main_Page'));
+  final request = await client.getUrl(
+    Uri.parse(
+        'https://api.wikimedia.org/feed/v1/wikipedia/$language/featured/$today'),
```

Comment:
Interesting. There's no API endpoint for just "most current news"? Is that `$today` forced by the API?

Reply:
Unfortunatelly there is no.